### ISSUE TITLE
OSError: Error no file named model_index.json found in directory /content/drive/MyDrive/stable_diffusion_weights/...

### ISSUE THREAD

**Issue by DjTecmo** [googlecolab/colabtools#3189]
Why ALWAYS I RUN THIS NOTEBOOK does the files accompanying with the respective .json file are missing on my weights directory and not created while running correctly the cells? The folder is empty and THIS OCCURS SO MANY TIMES EVERY DAY I RUN THE NOTEBOOK

**Comment by mayankmalik-colab** [comment]
We can have a look if you can share a minimal repro notebook.

**Comment by mco-gh** [comment]
Did you check the save_to_gdrive box under "Settings and run"? If not, you'll want to do that (note that is advertised as requiring 4-5 GB of space in Google Drive).

**Comment by mco-gh** [comment]
Once I checked that box, I was able to train the model with the resulting weights stored in the expected place:  
```
[*] Weights saved at /content/drive/MyDrive/stable_diffusion_weights/sks/800
```
and I managed to get past the Inference section where you were having problems. 

I also noticed that I had to upload an image in the cell starting with "Upload your images by running this cell.".

I'm guessing either a) you didn't check the save_to_gdrive box, b) you didn't upload an image, or c) there's some other environmental factor causing this to fail for you.

If you check the save_to_gdrive box and uploaded an image and it still doesn't work for you, I'd suggest filing an issue with the notebook author.
